pkg/logger/loggerinterface.go

```diff
@@ -0,0 +1,64 @@
+package logger
+
+import "time"
+
+// LogSave is the main system to specify
+// how to save log information
+type LogSave interface {
+	SaveLogData(message string)
+}
+
+// Logger is the main interface struct
+// for different types of loggin
+type Logger struct {
+	LogSave LogSave
+}
+
+// SystemLogger is log data directly from
+// the underlying running system
+type SystemLogger Logger
+
+// SystemLogTag is the identifying name for
+// system messages in the log files
+const SystemLogTag = "SYSTEM_LOG_INFO"
+
+// SystemErrorTag is the identifying name for
+// system errors in the log files
+const SystemErrorTag = "SYSTEM_ERROR"
+
+const logTimeStamp = "Mon Jan _2 15:04:05 UTC 2006"
```

Comment:
That's how go parses timestamps apparently, the returned result would be Mon Jan 30 13:17:00 UTC 2017